packages/p2p/src/components/app.scss

```diff
@@ -11,19 +11,41 @@
     }
     & .dc-tabs {
         position: static;
+        &--deriv-p2p {
```

Comment:
same here, please keep it brand-agnostic

Reply:
Thank you 👍